src/genegraph/transform/clinvar/clinical_assertion.clj

```diff
@@ -308,33 +312,53 @@
                  :release-date release-date))
     (first rs)))
 
-(defn ^RDFResource proposition-object [event]
-  (let [message (:genegraph.transform.clinvar.core/parsed-value event)
-        assertion (:content message)
-        trait-set-id (:trait_set_id assertion)]
-    (if (not (nil? trait-set-id))
-      (let [trait-set-resource (get-trait-set-by-id trait-set-id (:release_date message))]
-        trait-set-resource)
-      (log/warn :fn :proposition-object :msg :nil-trait-set-id :message message))))
+#_(defn ^RDFResource proposition-object [event]
+    (let [message (:genegraph.transform.clinvar.core/parsed-value event)
+          assertion (:content message)
+          trait-set-id (:trait_set_id assertion)]
+      (if (not (nil? trait-set-id))
+        (let [trait-set-resource (get-trait-set-by-id trait-set-id (:release_date message))]
+          trait-set-resource)
+        (log/warn :fn :proposition-object :msg :nil-trait-set-id :message message))))
 
+;; TODO This function is slower than it should be. Look into this.
 (defn variation-descriptor-by-clinvar-id [clinvar-id release-date]
-  (log/info :fn :variation-descriptor-by-clinvar-id
-            :clinvar-id clinvar-id
-            :release-date release-date)
+  (log/debug :fn :variation-descriptor-by-clinvar-id
+             :clinvar-id clinvar-id
+             :release-date release-date)
   (let [qualified-id (str iri/clinvar-variation clinvar-id)
+
+        _  (comment (q/select (str
+                               "select ?i where { \n"
+                               "?i a :vrs/CanonicalVariationDescriptor . \n"
+                               "?i :vrs/extensions ?ext . \n"
+                               "#?ext :vrs/name \"clinvar_variation\" . \n"
```

Comment:
Flagging this sparql line, along with the TODO comment below, related to performance.